### PR TITLE
Pull updates from ooi-integration to WHOIGit/ooicgsn (master)

### PR DESCRIPTION
**Pull request by snwhite**

### CODE REVIEW

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -2527,6 +2531,8 @@ OL000553,OL000553,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instrument
 OL000563,OL000563,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,CP03ISSM-00001-MOPAK,,,,CP03ISSM-00001-MOPAK,
 OL000564,OL000564,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,CP03ISSM-00001-SBD11-HYDGN,,,,CP03ISSM-00001-SBD11-HYDGN,
 OL000565,OL000565,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,CP03ISSM-00001-SBD12-HYDGN,,,,CP03ISSM-00001-SBD12-HYDGN,
+PIRSN-AOABPA-00001,,Sensor,0,,,Wilcock,,1,,,,,
+PIRSN-CAMPIA-00001,,Sensor,0,,,MARUM,,1,,,,,
 PIRSN-COVISA-00001,,Sensor,0,Cabled Observatory Vent Imaging System,COVIS Series A,Bemis,,1,,,,,
 PIRSN-CTDPFA-10001,,Sensor,0,CTD with optode,CTDPFA Series A wth optode,MARUM,16plusV2 with SBE 63 optode,16-50216,,,,,
 PIRSN-OVRSRA-00001,,Sensor,0,Overview Sonar,OVRSR Series A,MARUM,R2 Sonic 2022,101409,,,,,
```
Comment (reedan88) [inline bulk/sensor_bulk_load-AssetRecord.csv]: Reviewed.

File: cruise/CruiseInformation.csv [hunk 2]
```diff
@@ -160,4 +160,5 @@ OC1905C,R/V Oceanus,2019-05-03T20:00:00,2019-05-08T00:00:00,Endurance 11B
 BK20190504,M/V Blackhawk,2019-05-04T09:00:00,2019-05-04T20:00:00,Pioneer cp_340 recovery
 TS20190516,F/V Tenacious II,2019-05-16T12:00:00,2019-05-17T12:00:00,Pioneer 12a glider deployment
 SKQ201913T,R/V Sikuliaq,2019-05-14T00:00:00,2019-05-27T00:00:00,Papa Glider gp_469 recovery on the San Diego - Seward transit
-EK20190529,R/V Elakha,2019-05-29T16:22:00,2019-05-29T19:33:00,ce01issp_00013 deployment
\ No newline at end of file
+EK20190529,R/V Elakha,2019-05-29T16:22:00,2019-05-29T19:33:00,ce01issp_00013 deployment
+AT-42-12,R/V Atlantis,2019-06-02T00:00:00,2019-07-09T00:00:00,Cabled (rvdata) Visions 19
```
Comment (reedan88) [inline cruise/CruiseInformation.csv]: Reviewed.

File: deployment/CE02SHBP_Deploy.csv [hunk 3]
```diff
@@ -42,14 +42,25 @@ RR1717,,,,CE02SHBP-LJ01D-08-OPTAAD106,4,1,2017-09-10T14:30:00,2018-06-29T02:30:0
...
+AT-42-12,,,,CE02SHBP-LJ01D-10-PHSEND103,6,1,2019-07-25T00:00:00,,ATAPL-65310-820-0009,,ATOSU-70571-00004,44.63708,-124.30595,,80,80,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,CE02SHBP-LJ01D-11-HYDBBA106,6,1,2019-07-25T00:00:00,,ATAPL-65310-820-0009,,ATOSU-58324-00012,44.63704,-124.30586,,81,81,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,CE02SHBP-MJ01C-07-ZPLSCB101,5,1,2019-07-25T00:00:00,,ATAPL-65244-030-0025,,ATOSU-63259-00001,44.63739,-124.30547,,81,81,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,CE02SHBP-MJ01C-08-CAMDSB107,6,1,2019-07-25T00:00:00,,ATAPL-65244-030-0025,,ATAPL-58317-00002,44.6373,-124.30549,,80,80,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+
```
Comment (reedan88) [inline deployment/CE02SHBP_Deploy.csv]: Reviewed.

File: deployment/CE04OSBP_Deploy.csv [hunk 4]
```diff
@@ -33,12 +33,21 @@ RR1714,,,,CE04OSBP-LJ01C-08-OPTAAC104,4,1,2017-08-10T16:00:00,2018-06-25T00:00:0
...
+AT-42-12,,,,CE04OSBP-LJ01C-09-PCO2WB104,6,1,2019-07-25T00:00:00,,ATAPL-65310-810-0008,,ATOSU-70570-00003,44.3695,-124.95369,,581,581,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,CE04OSBP-LJ01C-10-PHSEND107,6,1,2019-07-25T00:00:00,,ATAPL-65310-810-0008,,ATOSU-70571-00003,44.3695,-124.95369,,581,581,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,CE04OSBP-LJ01C-11-HYDBBA105,6,1,2019-07-25T00:00:00,,ATAPL-65310-810-0008,,ATOSU-58324-00013,44.3695,-124.95369,,581,581,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,CE04OSBP-LV01C-06-CAMDSB106,6,1,2019-07-25T00:00:00,,ATAPL-65291-030-0045,,ATAPL-58317-00001,44.3693,-124.9541,,582,582,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
```
Comment (reedan88) [inline deployment/CE04OSBP_Deploy.csv]: Reviewed.

File: deployment/CE04OSPD_Deploy.csv [hunk 5]
```diff
@@ -16,8 +16,14 @@ SKQ201610S,,,,CE04OSPD-DP01B-02-VEL3DA105,3,1,2016-08-07T03:50:00,2018-07-23T00:
...
+AT-42-12,,,,CE04OSPD-DP01B-02-VEL3DA105,5,1,2019-07-25T00:00:00,,ATAPL-71403-00005,,ATAPL-58346-00005,44.36829,-124.9528,,nan,583,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,CE04OSPD-DP01B-03-FLCDRA103,5,1,2019-07-25T00:00:00,,ATAPL-71403-00005,,ATOSU-70111-00006,44.36829,-124.9528,,nan,583,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,CE04OSPD-DP01B-04-FLNTUA103,5,1,2019-07-25T00:00:00,,ATAPL-71403-00005,,ATAPL-70110-00005,44.36829,-124.9528,,nan,583,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,CE04OSPD-DP01B-06-DOSTAD105,5,1,2019-07-25T00:00:00,,ATAPL-71403-00005,,ATAPL-58320-00015,44.36829,-124.9528,,nan,583,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+
```
Comment (reedan88) [inline deployment/CE04OSPD_Deploy.csv]: Reviewed.

File: deployment/CE04OSPS_Deploy.csv [hunk 6]
```diff
@@ -59,18 +59,34 @@ RR1713,,,,CE04OSPS-SF01B-3D-SPKIRA102,4,1,2017-07-29T05:00:00,2018-07-17T15:00:0
...
+AT-42-12,,,,CE04OSPS-SF01B-3D-SPKIRA102,6,1,2019-07-25T00:00:00,,ATAPL-68870-002-0144,,ATOSU-58341-00008,44.37415,-124.95648,,nan,580,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,CE04OSPS-SF01B-4A-NUTNRA102,6,1,2019-07-25T00:00:00,,ATAPL-68870-002-0144,,ATOSU-68020-00008,44.37415,-124.95648,,nan,580,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,CE04OSPS-SF01B-4B-VELPTD106,6,1,2019-07-25T00:00:00,,ATAPL-68870-002-0144,,ATOSU-70114-10008,44.37415,-124.95648,,nan,580,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,CE04OSPS-SF01B-4F-PCO2WA102,6,1,2019-07-25T00:00:00,,ATAPL-68870-002-0144,,ATOSU-58336-00005,44.37415,-124.95648,,nan,580,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+
```
Comment (reedan88) [inline deployment/CE04OSPS_Deploy.csv]: Reviewed.

File: deployment/RS01SBPD_Deploy.csv [hunk 7]
```diff
@@ -20,8 +20,14 @@ SKQ201610S,,,,RS01SBPD-DP01A-02-VEL3DA103,3,1,2016-08-09T03:16:00,2018-07-25T00:
...
+AT-42-12,,,,RS01SBPD-DP01A-02-VEL3DA103,5,1,2019-07-25T00:00:00,,ATAPL-71403-00002,,ATAPL-58346-00003,44.52757,-125.38075,,nan,2903,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS01SBPD-DP01A-03-FLCDRA102,5,1,2019-07-25T00:00:00,,ATAPL-71403-00002,,ATAPL-70111-00002,44.52757,-125.38075,,nan,2903,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS01SBPD-DP01A-04-FLNTUA102,5,1,2019-07-25T00:00:00,,ATAPL-71403-00002,,ATAPL-70110-00002,44.52757,-125.38075,,nan,2903,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS01SBPD-DP01A-06-DOSTAD104,5,1,2019-07-25T00:00:00,,ATAPL-71403-00002,,ATAPL-58320-00001,44.52757,-125.38075,,nan,2903,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+
```
Comment (reedan88) [inline deployment/RS01SBPD_Deploy.csv]: Reviewed.

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 1 above)
Comment (cdobs) [inline bulk/sensor_bulk_load-AssetRecord.csv]: Looks okay

File: cruise/CruiseInformation.csv (hunk 2 above)
Comment (cdobs) [inline cruise/CruiseInformation.csv]: Looks okay

File: deployment/CE02SHBP_Deploy.csv (hunk 3 above)
Comment (cdobs) [reply deployment/CE02SHBP_Deploy.csv]: Looks okay

File: deployment/CE04OSBP_Deploy.csv (hunk 4 above)
Comment (cdobs) [reply deployment/CE04OSBP_Deploy.csv]: Looks okay

File: deployment/CE04OSPD_Deploy.csv (hunk 5 above)
Comment (cdobs) [reply deployment/CE04OSPD_Deploy.csv]: Looks okay

File: deployment/CE04OSPS_Deploy.csv (hunk 6 above)
Comment (cdobs) [reply deployment/CE04OSPS_Deploy.csv]: Looks okay

File: deployment/RS01SBPD_Deploy.csv (hunk 7 above)
Comment (cdobs) [reply deployment/RS01SBPD_Deploy.csv]: Looks okay

File: deployment/RS01SBPS_Deploy.csv [hunk 8]
```diff
@@ -91,13 +91,32 @@ RR1809,,,,RS01SBPS-SF01A-3D-SPKIRA101,5,1,2018-06-24T02:00:00,2018-07-09T12:00:0
...
+AT-42-12,,,,RS01SBPS-SF01A-3D-SPKIRA101,7,1,2019-07-25T00:00:00,,ATAPL-68870-001-0143,,ATAPL-58341-00006,44.52897,-125.38966,,nan,2906,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS01SBPS-SF01A-4A-NUTNRA101,7,1,2019-07-25T00:00:00,,ATAPL-68870-001-0143,,ATAPL-68020-00006,44.52897,-125.38966,,nan,2906,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS01SBPS-SF01A-4B-VELPTD102,7,1,2019-07-25T00:00:00,,ATAPL-68870-001-0143,,ATAPL-70114-00006,44.52897,-125.38966,,nan,2906,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS01SBPS-SF01A-4F-PCO2WA101,7,1,2019-07-25T00:00:00,,ATAPL-68870-001-0143,,ATAPL-58336-00003,44.52897,-125.38966,,nan,2906,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+
```
Comment (cdobs) [inline deployment/RS01SBPS_Deploy.csv]: Looks okay

File: deployment/RS01SLBS_Deploy.csv [hunk 9]
```diff
@@ -23,12 +23,18 @@ SKQ201610S,,,,RS01SLBS-LJ01A-11-OPTAAC103,3,1,2016-07-17T02:10:00,2017-07-29T04:
...
+AT-42-12,,,,RS01SLBS-LJ01A-10-ADCPTE101,5,1,2019-07-25T00:00:00,,ATAPL-65310-830-0010,,ATAPL-68073-00003,44.51519,-125.38987,,2902,2902,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS01SLBS-LJ01A-11-OPTAAC103,6,1,2019-07-25T00:00:00,,ATAPL-65310-830-0010,,ATAPL-69943-00002,44.51524,-125.38981,,2903,2903,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS01SLBS-LJ01A-12-CTDPFB101,6,1,2019-07-25T00:00:00,,ATAPL-65310-830-0010,,ATAPL-67627-00004,44.51524,-125.38981,,2903,2903,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS01SLBS-LJ01A-12-DOSTAD101,6,1,2019-07-25T00:00:00,,ATAPL-65310-830-0010,,ATAPL-58320-00013,44.51524,-125.38981,,2903,2903,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+
```
Comment (cdobs) [inline deployment/RS01SLBS_Deploy.csv]: Okay

File: deployment/RS01SUM1_Deploy.csv [hunk 10]
```diff
@@ -13,3 +13,5 @@ SKQ201610S,,,,RS01SUM1-LJ01B-12-VEL3DB104,2,1,2016-07-24T04:21:00,2017-08-11T00:
 RR1714,,,,RS01SUM1-LJ01B-09-PRESTB102,3,1,2017-08-11T23:00:00,,ATAPL-65310-850-0012,,ATAPL-69923-00002,44.56927,-125.14804,,775,775,
 RR1714,,,,RS01SUM1-LJ01B-12-VEL3DB104,3,1,2017-08-11T23:00:00,2018-06-26T03:30:00,ATAPL-65310-850-0012,,ATAPL-67979-00004,44.56911,-125.14797,,775,775,
 RR1809,,,,RS01SUM1-LJ01B-12-VEL3DB104,4,1,2018-06-26T17:30:00,,ATAPL-65310-850-0012,,ATAPL-67979-00002,44.56911,-125.14797,,775,775,
+AT-42-12,,,,RS01SUM1-LJ01B-06-OBSSPA103,2,1,2019-07-25T00:00:00,,ATAPL-65310-020-0004,,ATAPL-58331-00011,44.56739,-125.14418,,789,789,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+
```
Comment (cdobs) [inline deployment/RS01SUM1_Deploy.csv]: Okay

File: deployment/RS01SUM2_Deploy.csv [hunk 11]
```diff
@@ -7,16 +7,23 @@ TN-313,,,,RS01SUM2-MJ01B-05-CAMDSB103,1,1,2014-08-31T19:17:45,2016-07-22T00:00:0
...
+AT-42-12,,,,RS01SUM2-MJ01B-00-OSMOIA101,3,1,2019-07-25T00:00:00,,ATAPL-65244-020-0024,,ATAPL-58333-00003,44.34199,-125.08807,,774,774,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS01SUM2-MJ01B-05-CAMDSB103,5,1,2019-07-25T00:00:00,,ATAPL-65244-020-0024,,ATAPL-58317-00010,44.56996,-125.14686,,774,774,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS01SUM2-MJ01B-08-OVRSRA101,2,1,2019-07-25T00:00:00,,ATAPL-65244-020-0024,,PIRSN-OVRSRA-00001,44.57,-125.14671,,774,774,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS01SUM2-MJ01B-09-CAMPIA101,1,1,2019-07-25T00:00:00,,ATAPL-65244-020-0024,,PIRSN-CAMPIA-00001,44.57,-125.14681,,775,775,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+
```
Comment (cdobs) [inline deployment/RS01SUM2_Deploy.csv]: Okay

File: deployment/RS03ASHS_Deploy.csv [hunk 12]
```diff
@@ -4,14 +4,19 @@ TN-299,,,,RS03ASHS-MJ03B-05-OBSSPA302,1,1,2013-08-11T13:52:00,,ATAPL-65244-820-0
...
+AT-42-12,,,,RS03ASHS-MJ03B-00-OSMOIA301,3,1,2019-07-25T00:00:00,,ATAPL-65244-820-0033,,ATAPL-58333-00004,45.93367,-130.01369,,1541,1541,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS03ASHS-MJ03B-10-CTDPFB304,5,1,2019-07-25T00:00:00,,ATAPL-65244-820-0033,,ATCWK-67627-00006,45.93363,-130.01368,,1542,1542,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS03ASHS-MJ03B-12-COVISA301,2,1,2019-07-25T00:00:00,,ATAPL-65244-820-0033,,PIRSN-COVISA-00001,45.93363,-130.01368,,1542,1542,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS03ASHS-PN03B-06-CAMHDA301,6,1,2019-07-25T00:00:00,,ATSOO-70232-01H3B,,ATAPL-70248-00002,45.93362,-130.01364,,1542,1542,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+
```
Comment (cdobs) [inline deployment/RS03ASHS_Deploy.csv]: Okay

File: deployment/RS03AXBS_Deploy.csv [hunk 13]
```diff
@@ -28,7 +28,12 @@ RR1713,,,,RS03AXBS-LJ03A-10-ADCPTE303,4,1,2017-07-31T22:30:00,,ATAPL-65310-840-0
...
+AT-42-12,,,,RS03AXBS-LJ03A-05-HPIESA301,5,1,2019-07-25T00:00:00,,ATAPL-65310-840-0011,,ATAPL-58323-00001,45.82405,-129.75995,,2609,2609,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS03AXBS-LJ03A-11-OPTAAC303,6,1,2019-07-25T00:00:00,,ATAPL-65310-840-0011,,ATAPL-69943-00009,45.81688,-129.75425,,2606,2606,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS03AXBS-LJ03A-12-CTDPFB301,6,1,2019-07-25T00:00:00,,ATAPL-65310-840-0011,,ATAPL-67627-00005,45.81688,-129.75425,,2606,2606,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS03AXBS-LJ03A-12-DOSTAD301,6,1,2019-07-25T00:00:00,,ATAPL-65310-840-0011,,ATAPL-58320-00014,45.81688,-129.75425,,2606,2606,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+
```
Comment (cdobs) [inline deployment/RS03AXBS_Deploy.csv]: Okay

File: deployment/RS03AXPD_Deploy.csv [hunk 14]
```diff
@@ -11,8 +11,14 @@ RR1715,,,,RS03AXPD-DP03A-02-VEL3DA303,2,1,2017-08-23T09:30:00,2018-07-26T00:00:0
...
+AT-42-12,,,,RS03AXPD-DP03A-02-VEL3DA303,4,1,2019-07-25T00:00:00,,ATAPL-71403-00001,,ATAPL-58346-00006,45.82972,-129.75904,,nan,2901,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS03AXPD-DP03A-03-FLCDRA302,4,1,2019-07-25T00:00:00,,ATAPL-71403-00001,,ATAPL-70111-00005,45.82972,-129.75904,,nan,2901,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS03AXPD-DP03A-03-FLNTUA302,4,1,2019-07-25T00:00:00,,ATAPL-71403-00001,,ATOSU-70110-00006,45.82972,-129.75904,,nan,2901,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS03AXPD-DP03A-06-DOSTAD304,4,1,2019-07-25T00:00:00,,ATAPL-71403-00001,,ATOSU-58320-00018,45.82972,-129.75904,,nan,2901,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+
```
Comment (cdobs) [inline deployment/RS03AXPD_Deploy.csv]: Okay

File: deployment/RS03AXPS_Deploy.csv [hunk 15]
```diff
@@ -69,3 +69,22 @@ RR1713,,,,RS03AXPS-SF03A-3D-SPKIRA301,4,1,2017-07-31T04:00:00,2018-07-02T20:30:0
...
+AT-42-12,,,,RS03AXPS-SF03A-3D-SPKIRA301,5,1,2019-07-25T00:00:00,,ATAPL-68870-001-0145,,ATAPL-58341-00007,45.83049,-129.75326,,nan,2609,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS03AXPS-SF03A-4A-NUTNRA301,5,1,2019-07-25T00:00:00,,ATAPL-68870-001-0145,,ATAPL-68020-00007,45.83049,-129.75326,,nan,2609,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS03AXPS-SF03A-4B-VELPTD302,5,1,2019-07-25T00:00:00,,ATAPL-68870-001-0145,,ATAPL-70114-00007,45.83049,-129.75326,,nan,2609,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS03AXPS-SF03A-4F-PCO2WA301,5,1,2019-07-25T00:00:00,,ATAPL-68870-001-0145,,ATAPL-58336-00009,45.83049,-129.75326,,nan,2609,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+
```
Comment (cdobs) [inline deployment/RS03AXPS_Deploy.csv]: Okay

File: vocab/vocab.csv [hunk 16]
```diff
@@ -1452,6 +1452,7 @@ RS03CCAL-MJ03F,Cabled Axial Seamount,Central Caldera,Medium-Power JBox (MJ03F),,
 RS03CCAL-MJ03F-05-BOTPTA301,Cabled Axial Seamount,Central Caldera,Medium-Power JBox (MJ03F),Bottom Pressure and Tilt,PMEL/Chadwick,PMELcabled BPR/Tilt,1526,1526
 RS03CCAL-MJ03F-06-HYDLFA305,Cabled Axial Seamount,Central Caldera,Medium-Power JBox (MJ03F),Low Frequency Acoustic Receiver (Hydrophone),HTI,90-U,1526,1526
 RS03CCAL-MJ03F-06-OBSBBA301,Cabled Axial Seamount,Central Caldera,Medium-Power JBox (MJ03F),Broadband Ocean Bottom Seismometer,Guralp,CMG-1T/5T,1526,1526
+RS03CCAL-MJ03F-07-AOABPA301,Cabled Axial Seamount,Central Caldera,Medium-Power JBox (MJ03F),A-O-A Pressure Sensor,PI Wilcock,,1526,1526
 RS03CCAL-MJ03F-08-SCTAAA301,Cabled Axial Seamount,Central Caldera,Medium-Power JBox (MJ03F),Self-Calibrating Triaxial Accelerometer,PI Wilcock,SCTA Series A,1526,1526
 RS03CCAL-MJ03F-09-SCPRAA301,Cabled Axial Seamount,Central Caldera,Medium-Power JBox (MJ03F),Self-Calibrating Pressure Recorder,PI Zumberge and Sasagawa,SCPR Series A,1526,1526
 RS03ECAL,Cabled Axial Seamount,Eastern Caldera,,,,,1516,1516
```
Comment (cdobs) [inline vocab/vocab.csv]: Okay

File: deployment/RS03INT1_Deploy.csv [hunk 17]
```diff
@@ -24,9 +24,14 @@ RR1715,,,,RS03INT1-MJ03C-07-PPSDNA301,4,1,2017-08-22T10:30:00,2018-07-04T09:30:0
...
+AT-42-12,,,,RS03INT1-MJ03C-07-PPSDNA301,6,1,2019-07-25T00:00:00,,ATAPL-65244-060-0028,,ATAPL-58338-00003,45.92617,-129.97897,,1521,1521,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS03INT1-MJ03C-07-RASFLA301,6,1,2019-07-25T00:00:00,,ATAPL-65244-060-0028,,ATAPL-58340-00002,45.92617,-129.97897,,1521,1521,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS03INT1-MJ03C-09-THSPHA301,6,1,2019-07-25T00:00:00,,ATAPL-65244-060-0028,,ATAPL-58342-00002,45.92637,-129.97905,,1520,1520,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+AT-42-12,,,,RS03INT1-MJ03C-10-TRHPHA301,5,1,2019-09-01T00:00:00,,ATAPL-65244-060-0028,,ATAPL-67653-00002,45.92636,-129.97921,,1520,1520,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
```
Comment (cdobs) [inline deployment/RS03INT1_Deploy.csv]: Okay

File: deployment/RS03CCAL_Deploy.csv [hunk 18]
```diff
@@ -4,3 +4,5 @@ TN-313,,,,RS03CCAL-MJ03F-06-HYDLFA305,1,1,2014-07-26T20:43:24,,ATAPL-65244-090-0
 TN-313,,,,RS03CCAL-MJ03F-06-OBSBBA301,1,1,2014-07-26T20:43:24,,ATAPL-65244-090-0031,,ATAPL-58328-00003,45.95468,-130.00893,,1528,1528,
 RR1810,,,,RS03CCAL-MJ03F-08-SCTAAA301,1,1,2018-07-06T07:00:00,,ATAPL-65244-010-0023,,PIRSN-SCTAAA-00001,45.95468,-130.00893,,1528,1528,
 RR1810,,,,RS03CCAL-MJ03F-09-SCPRAA301,1,1,2018-07-06T07:00:00,,ATAPL-65244-010-0023,,PIRSN-SCPRAA-00001,45.95468,-130.00893,,1528,1528,
+AT-42-12,,,,RS03CCAL-MJ03F-07-AOABPA301,1,1,2019-07-25T00:00:00,,ATAPL-65244-010-0023,,PIRSN-AOABPA-00001,45.95468,-130.00893,,1528,1528,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth
+
```
Comment (cdobs) [inline deployment/RS03CCAL_Deploy.csv]: Okay

**Pull request merged**